services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/AutoRerouteV2Spec.groovy

```diff
@@ -441,6 +403,90 @@ class AutoRerouteV2Spec extends HealthCheckSpecification {
         database.resetCosts()
     }
 
+    @Tags(VIRTUAL)
+    def "Flow in 'Down' status is rerouted after switchUp event"() {
+        given: "Two active neighboring switches with two parallel links"
+        def switchPair = topologyHelper.getAllNeighboringSwitchPairs().find {
+            it.paths.findAll { it.size() == 2 }.size() > 1
+        } ?: assumeTrue("No suiting switches found", false)
+
+        and: "Main and backup paths for further manipulation with them"
```

Comment:
Please don't get too keen on _commenting the code_ whith given/when/then. These blocks must describe the _test case_, not the code. You can comment code using regular `//comment` comments